src/pages/NavBar.tsx

```diff
@@ -252,6 +242,7 @@ function GetStarted() {
 }
 
 const before_pages = ['home', 'about', 'learn', 'solve']
+// TODO(SATH): After login navbar change
 // const after_pages = ['home', 'about', 'learn', 'solve', 'submit'];
```

Comment:
so what is this?

Reply:
We wanted different navbar options when user is logged in (submit for judge with user's own solution and I/O files)